cpu/efm32/cpu.c

```diff
@@ -106,13 +107,11 @@ static void clk_init(void)
 #endif
 
     /* initialize LFXO with board-specific parameters before switching */
-    if (CLOCK_LFA == cmuSelect_LFXO || CLOCK_LFB == cmuSelect_LFXO ||
+    if (CLOCK_LFA == cmuSelect_LFXO || CLOCK_LFB == cmuSelect_LFXO
 #if defined(_SILICON_LABS_32B_SERIES_1)
-        CLOCK_LFE == cmuSelect_LFXO)
-#else
-        false)
+        || CLOCK_LFE == cmuSelect_LFXO
 #endif
-    {
+    ) {
```

Comment:
It might be better readable to use rather something like
```diff
-     if (CLOCK_LFA == cmuSelect_LFXO || CLOCK_LFB == cmuSelect_LFXO
- #if defined(_SILICON_LABS_32B_SERIES_1)
-         || CLOCK_LFE == cmuSelect_LFXO
- #endif
-     ) {
+ #if defined(_SILICON_LABS_32B_SERIES_1)
+     if (CLOCK_LFA == cmuSelect_LFXO || CLOCK_LFB == cmuSelect_LFXO || CLOCK_LFE == cmuSelect_LFXO) {
+ #else
+     if (CLOCK_LFA == cmuSelect_LFXO || CLOCK_LFB == cmuSelect_LFXO) {
+ #endif
```
Even GitHub has problems to parse this construct correctly. That's why I had to use diff instead of suggestion.

Reply:
I don't know whether vera++ likes this.

Reply:
Please squash.